src/main/java/woowacourse/common/ui/CommonControllerAdvice.java

```diff
@@ -0,0 +1,36 @@
+package woowacourse.common.ui;
+
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import woowacourse.common.exception.AuthException;
+import woowacourse.common.exception.CustomException;
+import woowacourse.common.exception.InputFormatException;
+import woowacourse.common.exception.JoinException;
+import woowacourse.common.exception.LoginException;
+import woowacourse.common.exception.dto.ErrorResponse;
+
+@RestControllerAdvice
+public class CommonControllerAdvice {
+    @ExceptionHandler({LoginException.class, JoinException.class, InputFormatException.class})
```

Comment:
```suggestion
    @ExceptionHandler({CustomException.class})
```

커스텀 예외에 대해서 일괄적으로 처리하면 예외가 추가될 때마다 ControllerAdvice가 변경되는 걸 피할 수 있을 것 같아요.

Reply:
해당 커스텀 에러의 경우 AuthException에서 Unauthorized 에러코드를, 해당 Exception을 제외한 나머지에서는 Bad Request 에러코드를 만들어서 각각의 내용을 따로 기입하였습니다.